Backend/controllers/productservices.js

```diff
@@ -0,0 +1,80 @@
+import productService from "../models/productservices";
+/**
+ * @module controllers/productservices.js
+ * @requires models/productServices.js
+ * @description Create a new product or service in the database
+ * 
+ */
+
+export const productServiceSchema = async (req, res) => {
```

Comment:
You may need to rename this function to `createProductService` instead of `productServiceSchema` so that it can be descriptive of the what the function does

Reply:
Okay,  I'll do that . Thanks